routes/web.php

```diff
@@ -13,4 +13,17 @@
 
 Auth::routes();
 
+//link to dashboard/home
 Route::get('/', 'HomeController@index')->name('dashboard');
+
+//link to Roles Page
+Route::get('/roles', 'RoleController@index')->name('roles');
+
+//link to Add Role Page
+Route::get('/roles/add', 'RoleController@create')->name('rolesAdd');
+
+//action that stores role
+Route::post('/add', 'RoleController@store')->name('roleStore');
```

Comment:
@jeremiahC Let's use `.` as delimeter for route name, e.g `roleStore` -> `role.store`